File: RentalAttireBackend/RentalAttireBackend/Application/Common/Models/AuthenticationResult.cs
using RentalAttireBackend.Application.Users.DTO;

namespace RentalAttireBackend.Application.Common.Models
{
    public class AuthenticationResult
    {
        public string AccessToken { get; set; } = string.Empty;
        public string RefreshToken { get; set; } = string.Empty;
        public DateTime ExpiresAt { get; set; }
        public int Id { get; set; }
        public string Email { get; set; } = string.Empty;
        public bool IsProfileComplete { get; set; } = false;
    }
}

File: RentalAttireFrontend/src/app/data/models/Results/authentication-result.ts
import { UserDTO } from "../DTOs/Users/user-dto";


export class AuthenticationResult {
    accessToken : string = '';
    refreshToken : string = '';
    expiresAt : Date = new Date();
    id: number = 0;
    email: string = '';
    isProfileComplete: boolean = false;
}